dev/src/types.ts

```diff
@@ -20,6 +20,123 @@ import {Timestamp} from './timestamp';
 
 import api = google.firestore.v1;
 
+/*!
```

Comment:
This block is moved from watch.ts. I eventually want to use the error code definition included in Google Gax.

Reply:
Gosh, those status codes spread everywhere.

In `google-gax`, we already have `src/status.ts` that basically copies the error codes enum from gRPC. It's not exported though. If we export it from gax, we can reuse it here.

Reply:
I am hoping that https://github.com/googleapis/gax-nodejs/pull/693 will solve this.

Reply:
https://github.com/googleapis/gax-nodejs/pull/693 is merged and a new version will be released on January 2nd. Thank you!

Reply:
Can you also merge https://github.com/googleapis/gax-nodejs/pull/695? Thanks!

Reply:
Done! Release is expected to be done tomorrow in the morning.